tools/llvm2bpl/llvm2bpl.cpp

```diff
@@ -122,6 +123,7 @@ int main(int argc, char **argv) {
   pass_manager.add(new llvm::MergeArrayGEP());
   // pass_manager.add(new smack::SimplifyLibCalls());
   pass_manager.add(new llvm::Devirtualize());
+  pass_manager.add(new smack::MemoryAllocationChecker());
```

Comment:
@zvonimir  Should I choose reuse or no-reuse as memory model when memory safety is enabled? if no memory model is chosen then the default one in impl which doesn't check memory safety